styles.css

```diff
@@ -53,8 +53,8 @@
   width: 100%;
   padding: 1rem;
   z-index: 2;
-  color:var(--white);
-  display:flex;
+  color: var(--white);
```

Comment:
just a suggestion, you can give variable name as "--white-color" for better understanding